src/liquidFillView.js

```diff
@@ -131,7 +131,7 @@ echarts.extendChartView({
                 }
 
                 var style = {};
-                var styleAttrs = ['fill', 'opacity', 'shadowBlur', 'shadowColor'];
+                var styleAttrs = ['color', 'fill', 'opacity', 'shadowBlur', 'shadowColor'];
```

Comment:
`itemStyle.color` maps to attribute `fill`, so adding `'color'` won't be necessary.